service-compose/fcp-sfd-comms.yaml

```diff
@@ -0,0 +1,29 @@
+services:
+  fcp-sfd-comms:
+    build:
+      context: ../../fcp-sfd-comms
+      target: development
+    image: fcp-sfd-comms-development
+    container_name: fcp-sfd-comms-development
+    ports:
+      - '3039:3039'
```

Comment:
this is the port for the wrong repo - see [confluence](https://eaflood.atlassian.net/wiki/spaces/SFD/pages/edit-v2/5032017926) for more information